Userland/Games/2048/main.cpp

```diff
@@ -151,14 +151,18 @@ int main(int argc, char** argv)
         case Game::MoveOutcome::InvalidMove:
             undo_stack.take_last();
             break;
-        case Game::MoveOutcome::Won:
+        case Game::MoveOutcome::Won: {
             update();
-            GUI::MessageBox::show(window,
-                String::formatted("You reached {} in {} turns with a score of {}", game.target_tile(), game.turns(), game.score()),
-                "You won!",
-                GUI::MessageBox::Type::Information);
-            start_a_new_game();
+            auto message_box = GUI::MessageBox::construct(window, "Congratulations! You won the game, Do you still want to continue?",
```

Comment:
The "<score> in <yay many moves>" message doesn't exist anymore, that doesn't seem right?

Reply:
Now that will get displayed when the board is stalled. But we can show user this information.

Reply:
I have fixed this issue.